packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -128,6 +139,8 @@ const styles = (theme: ThemeType) => ({
     color: theme.palette.text.alwaysWhite,
     fontSize: 34,
     borderRadius: '50%',
+    // offset the additional 5px on the left added by the larger font size
+    marginLeft: -5,
```

Comment:
I noticed that the placeholder version of the user image on the slider was slightly off-center